upload_function/app.py

```diff
@@ -45,7 +49,8 @@ def parse_s3_object_url(url_string) -> str:
 def parse_pathname(fullpath: str) -> object:
     sections = fullpath.split('/')
     if len(sections) != 4:
-        logger.error("Could not identify Application ID, Subscription ID, Question ID and filename from: %s", fullpath)
+        logger.error(
+            "Could not identify Application ID, Subscription ID, Question ID and filename from: %s", fullpath)
         return "", ""
```

Comment:
Appreciate this isn't your change but do you know why we are returning this instead of an empty string?

Reply:
Based on the other return statement of this function, guessing its because other methods expect this to return multiple strings